src/openzaak/components/zaken/openapi.yaml

```diff
@@ -1984,6 +2012,41 @@ paths:
                     type: array
                     items:
                       $ref: '#/components/schemas/Zaak'
+                  inclusions:
+                    type: object
+                    properties:
+                      catalogi:zaaktype:
+                        type: array
+                        items:
+                          $ref: https://raw.githubusercontent.com/VNG-Realisatie/catalogi-api/1.0.0/src/openapi.yaml#/components/schemas/ZaakType
```

Comment:
Hmm I see, how could I determine the list of existing minor versions for the components? Or should this be a list somewhere in `conf/api.py`?

Reply:
yeah, a custom setting in `conf/api.py` seems the simplest solution.

I think in general we might be looking at some more structured API information, e.g. something like:

```
CATALOGI_API = ApiMetadata(
    versions=[
        ApiVersion(version="1.0.1", oas="https://raw.githubusercontent.com/..."),
        ApiVersion(version="1.1.0", oas="https://raw.githubusercontent.com/..."),
    ]
)
```

But that can be part of another refactor - it would help with including the API specs at image build time instead of having to do network IO all the time (see #621)

Reply:
I added included support for this in the inspector (https://github.com/open-zaak/open-zaak/pull/1138/commits/072cca77c9095311107b980f4842d2f08a0d390d), when we actually do this API information refactor we can modify `get_external_schema_refs` to use these lists of supported API versions